README.md

```diff
@@ -23,3 +23,13 @@ EBean is a Java ORM library that uses SQL:
 and the documentation can be found here:
 
 [https://ebean-orm.github.io/](https://ebean-orm.github.io/)
+
+## Mysql configuration
```

Comment:
I don't see where that's specific to this example? If not would rather have a link yo the already existing doc about how to configure a DB

Reply:
The link is https://www.playframework.com/documentation/2.6.x/JavaDatabase,  if  we can write it here. It's more convenient for new user.

Reply:
Should we add this link to this document?

Reply:
As for other doc link there, if the one to the Play doc is not sufficient.

Reply:
ok